changes/1423.fix.md

```diff
@@ -0,0 +1 @@
+Enable to terminate a compute session when wsproxy server has any connection issue.
```

Comment:
The sentence looks strange. "has any connection issue"??

Reply:
Thanks for reviewing!
I will fix it.